### PR TITLE
[Siva, testplans] fixes for consistency

### PR DESCRIPTION
**Pull request by engdoreis**
This PR make some changes to the sival testplan in order to make them consistent

### CODE REVIEW

File: hw/ip/otbn/data/otbn.hjson [hunk 1]
```diff
@@ -213,16 +213,16 @@
         OTBN has a connection to the EDN allowing it to receive 256-bits of entropy.
       '''
       name: "OTBN.KEYMGR"
-      descr: '''
+      desc:'''
```
Comment (vogelpi) [inline hw/ip/otbn/data/otbn.hjson:216]: Nit: do you want to re-insert the whitespace between the colon and the `'`? Know you've replaced `descr: ` with `desc:`. I am not sure if the whitespace is needed though.

**Review by vogelpi — APPROVED** [review]
LGTM

File: hw/ip/otbn/data/otbn.hjson (hunk 1 above)
Comment (engdoreis) [reply hw/ip/otbn/data/otbn.hjson]: Good catch, the space is not needed for the tooling, but I added it back for consistency.

**Review by jwnrt — APPROVED** [review]

**Comment by jwnrt** [comment]
CHANGE AUTHORIZED: hw/ip/otbn/data/otbn.hjson
CHANGE AUTHORIZED: hw/ip/rv_core_ibex/data/rv_core_ibex.hjson
CHANGE AUTHORIZED: hw/top_earlgrey/ip_autogen/alert_handler/data/alert_handler.hjson

**Comment by engdoreis** [comment]
CHANGE AUTHORIZED: hw/ip/otbn/data/otbn.hjson
CHANGE AUTHORIZED: hw/ip/rv_core_ibex/data/rv_core_ibex.hjson
CHANGE AUTHORIZED: hw/top_earlgrey/ip_autogen/alert_handler/data/alert_handler.hjson

**Pull request merged**